packages/shared/src/lib/contexts/settings/actions/deleteProfile.ts

```diff
@@ -27,14 +23,15 @@ export async function deleteProfile(): Promise<void> {
          * CAUTION: The profile and its data must be removed from the
          * app's list of profiles that lives as a Svelte store.
          */
-        removeAllProfileData(_activeProfileId)
+        removeAllProfileData(profileId)
 
         /**
          * CAUTION: This removes the actual directory for the profile,
          * so it should occur last.
          */
-        await removeProfileFolder(_activeProfileId)
+        await removeProfileFolder(profileId)
 
+        closePopup()
```

Comment:
move this into the `ProfileCardActionsMenu` to keep this file `desktop` agnostic